dialects/postgresql/src/main/kotlin/app/cash/sqldelight/dialects/postgresql/grammar/mixins/DistinctOnExpressionMixin.kt

```diff
@@ -19,7 +21,13 @@ internal abstract class DistinctOnExpressionMixin(node: ASTNode) :
   private val distinctOnColumns get() = children.filterIsInstance<SqlResultColumn>()
 
   override fun queryAvailable(child: PsiElement): Collection<QueryElement.QueryResult> {
-    return (parent as SqlSelectStmt).queryExposed()
+    val distinctOnColumnsWithTablePrefix: List<NamedElement?> =
```

Comment:
Surely `mapNotNull` returns a non-null type parameter, right?
```suggestion
    val distinctOnColumnsWithTablePrefix: List<NamedElement> =
```
This should also avoid the need to `!!` inside the `flatMap`.

Reply:
Thank you - have fixed the unwanted nullability that was introduced